src/main/java/org/folio/rest/impl/FundDistributionsValidationHelper.java

```diff
@@ -43,7 +42,10 @@ public Future<Void> validateFundDistributions(ValidateFundDistributionsRequest r
         List<FundDistribution> fundDistributionList = request.getFundDistribution();
         if (CollectionUtils.isNotEmpty(request.getAdjustments())) {
           if (validator.isAdjustmentIdsNotUnique(request.getAdjustments())) {
-            throw new HttpException(400, ADJUSTMENT_IDS_NOT_UNIQUE);
+            var parameter = new Parameter().withKey("adjustments").withValue(request.getAdjustments().toString());
```

Comment:
getAdjustments has not toString() implementation, if you want to include it as param you need to use Json encode